tests/client/sdam/rs.rs

```diff
@@ -0,0 +1,18 @@
+use super::framework::run_suite;
+
+use std::fs;
+use std::path::Path;
+
+#[test]
+fn sdam_rs() {
+    let paths = fs::read_dir(&Path::new("tests/json/data/specs/source/server-discovery-and-monitoring/tests/rs/")).unwrap();
```

Comment:
It's from the `specifications` repo; we added it as a submodule back when we were implementing the CRUD spec tests. Assuming the user cloned recursively, it will always be in that relative path.